src/platform/forms/save-in-progress/ApplicationStatus.jsx

```diff
@@ -220,21 +217,19 @@ export class ApplicationStatus extends React.Component {
               .
             </p>
           )}
-          <Modal
-            cssClass="va-modal-large"
+          <VaModal
+            class="va-modal-large"
```

Comment:
The web component has an actual prop `large`. Is that what we meant to use here instead of the class? 
https://design.va.gov/storybook/?path=/docs/components-va-modal--large